src/gui/widgets/log.cc

```diff
@@ -41,6 +42,8 @@ void PCSX::Widgets::Log::draw(const char* title) {
     }
     ImGui::Checkbox(_("Follow"), &m_follow);
     ImGui::SameLine();
+    ImGui::Checkbox(_("Log CD-ROM"), &PCSX::g_emulator->settings.get<PCSX::Emulator::SettingLoggingCDROM>().value);
```

Comment:
Hmm.... I'm wondering if this is the appropriate place for this checkbox...

Maybe a menu?